features/orabos/file.include/usr/lib/dracut/modules.d/99ensure-hugepages/ensure-hugepages.sh

```diff
@@ -36,14 +36,19 @@ if [ $hugepages -le 0 ]; then
   exit 0
 fi
 
-cmdline="$(</proc/cmdline) hugepages=$hugepages"
-release=$(uname -r)
 
-NEWROOT=${NEWROOT:-/sysroot}
+echo $hugepages > /proc/sys/vm/nr_hugepages
+# might get more hugepages and faster with the kernel cmdline, but like this you avoid a kexec reboot.
```

Comment:
Just for my understanding: What exactly do you mean with this comment? Why do we avoid a kexec reboot when we use it like this?

Reply:
@gehoern moved applying hugepages to a slightly later point in booting meaning that the Linux kernel might move around some pages in that process (if than only a few at this moment in time).

Reply:
@NotTheEvilOne this part I understand, I just don't understand why we would need a kexec cycle otherwise.

Reply:
`kexec` has been used originally to apply the changes. This has been rewritten in that PR.

Reply:
Ah, then the comment makes sense. Thanks for elaborating.